examples/Bootstrap/package.json

```diff
@@ -29,7 +29,7 @@
     "react": "^16.14.0",
```

Comment:
bootstrap package must be update to 5.0.1

How did you test react-form-with-constraints-bootstrap without updating the example to Bootstrap 5?